graphics/librsvg/DEPENDS

```diff
@@ -1,16 +1,16 @@
 depends gdk-pixbuf
-depends pango      # needs pango with cairo enabled
+depends pango
 depends rustc
 depends intltool
 depends gtk-doc
 
 optional_depends vala \
                  "--enable-vala" \
-                 "--disable-vala" \
+                 "--enable-vala=no" \
```

Comment:
this is a standard autotools configure script, so `--disable-*` and `--enable-*=no` are equivalent. No reason to change this...